### PR TITLE
Developer CoP in a Fishbowl experiment

### PR DESCRIPTION
**Pull request by MerlinDarkstar**
Developer CoP in a Fishbowl experiment

### CODE REVIEW

File: content/engineering-practices-fishbowl-dev-cop.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Author: Martin M. Lacey
+Summary: Experimenting with a New Developer CoP Format: Fishbowl Discussions
+
+In this week's podcast, we're diving into a fresh approach for our Developer Community of Practice (Dev CoP). Over the years, we’ve leaned on standard presentations and open dialogue formats, but this time, we’re shaking things up with a fishbowl discussion. As part of our Scalable Agile Framework for the Enterprise (SAFe) practices at WorkSafeBC, this experiment aligns with our commitment to fostering innovation and engagement in our CoP ceremonies.
+
+# What Are Communities of Practice?
+
+Communities of Practice (CoPs) are groups of professionals united by a shared interest in a technical or business domain. They collaborate to exchange knowledge, refine their skills, and push the boundaries of their field. This concept, outlined by Etienne Wenger in Communities of Practice: Learning, Meaning, and Identity, emphasizes learning through collaboration and shared experiences.
+
+For more, see: Communities of Practice - Scaled Agile Framework.
```
Comment (wpschaub) [inline content/engineering-practices-fishbowl-dev-cop.md]: ... and https://wsbctechnicalblog.github.io/ceremony-community-of-practice.html? Self-advertising :)

**Review by wpschaub — DISMISSED** [review]

File: content/engineering-practices-fishbowl-dev-cop.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
+Date: 2023-12-14
+Category: Posts 
+Tags: engineering, technical-excellence, dev-cop, fishbowl
```
Comment (wsbctechnicalblog) [inline content/engineering-practices-fishbowl-dev-cop.md:4]: @MerlinDarkstar dev-cop and fishbowl are not valid tags. Will replace them with event when publishing.

**Review by wsbctechnicalblog — APPROVED** [review]

**Comment by AndreasNVI** [comment]
I guess my review was not required.... ;-)

**Review by wpschaub — APPROVED** [review]

File: content/engineering-practices-fishbowl-dev-cop.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
+Date: 2023-12-14
```
Comment (AndreasNVI) [inline content/engineering-practices-fishbowl-dev-cop.md:2]: You have the year as 2023?

File: content/engineering-practices-fishbowl-dev-cop.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+Title: Fishbowl Dev CoP
```
Comment (AndreasNVI) [inline content/engineering-practices-fishbowl-dev-cop.md:1]: When I go to the File View, I see this for this section - is this correct? 
image

**Review by vikasjawa — APPROVED** [review]

File: content/engineering-practices-fishbowl-dev-cop.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+# Adapting Fishbowl for Hybrid Teams
+
+Post-pandemic realities mean our teams are split between remote and in-office work, posing unique challenges for interactive formats like the fishbowl. To bridge this gap, we’re piloting a hybrid model. Picture this: the core participants—our “bowl”—will be on-site, surrounded by a physical audience. Simultaneously, virtual participants will join in via Microsoft Teams, forming a “virtual outer ring.” This hybrid setup allows everyone to engage, regardless of location.
+
+Our inaugural Dev CoP Fishbowl is scheduled for a day when many engineering teams, including mine, are in the office. Limited in-person seating will make attendance a coveted experience, promoting active participation. For those online, a large monitor will provide a closer view of the live interactions, ensuring inclusivity and collaboration.
+
+>
+> ![Fishbowl Format](../images/engineering-practices-fishbowl-dev-cop-1.png)
```
Comment (AndreasNVI) [inline content/engineering-practices-fishbowl-dev-cop.md:28]: Why does everyone in this image look as if they are under 30?

File: content/engineering-practices-fishbowl-dev-cop.md (hunk 3 above)
Comment (MerlinDarkstar) [reply content/engineering-practices-fishbowl-dev-cop.md:2]: good catch, thank you!

File: content/engineering-practices-fishbowl-dev-cop.md (hunk 4 above)
Comment (MerlinDarkstar) [reply content/engineering-practices-fishbowl-dev-cop.md:1]: Not sure, but I have fixed the date :)

File: content/engineering-practices-fishbowl-dev-cop.md (hunk 3 above)
Comment (wsbctechnicalblog) [reply content/engineering-practices-fishbowl-dev-cop.md:2]: @AndreasNVI the metadata is checked and fixed when we publish.

File: content/engineering-practices-fishbowl-dev-cop.md (hunk 5 above)
Comment (wsbctechnicalblog) [reply content/engineering-practices-fishbowl-dev-cop.md:28]: 'cause everyone else has retired to a remote island to chill :)

**Pull request merged**